### DISCUSSION TITLE
Difference between use (lever, button, door) and place

### DISCUSSION THREAD

**Discussion by maxsupermanhd** [adepierre/Botcraft#43] [Q&A]
Both actions are `ServerboundUseItemOnPacket` but how to differ placement of block from using a block?

**Comment by adepierre** [comment]
This is not an issue regarding botcraft, converting it to a discussion.

You can check the code for the two functions here, or alternatively check yourself using sniffcraft

https://github.com/adepierre/Botcraft/blob/5e2fbef45d54faa720d961b480acd040b1175782/botcraft/src/Game/InterfaceClient.cpp#L529

https://github.com/adepierre/Botcraft/blob/5e2fbef45d54faa720d961b480acd040b1175782/botcraft/src/Game/InterfaceClient.cpp#L670

**Reply by maxsupermanhd** [reply]
I did, I found no difference between them and that's why I am asking

**Reply by adepierre** [reply]
Well, so there is just no difference 😁

I guess it just depends on the coordinates sent.
